### ISSUE TITLE
docs: changelog of v0.3 vs v0.2

### ISSUE THREAD

**Issue by tikikun** [homebrewltd/ichigo#76]
### Problem Statement

Currently there is no different log of v0.3 and v0.2 we should have to show the different 

### Idea

Create such log (with table compare 1 by 1 different in phase etc) to have a good checklist + to annouce the release better

**Comment by bachvudinh** [comment]
## Version Comparison: v0.2 vs v0.3

| Feature | v0.2 | v0.3 | Changes |
|---------|------|------|---------|
| Data    |      |      |         |
| Training|      |      |         |
| Phase 1 |      |      |         |
| Phase 2 |      |      |         |
| Phase 3 |      |      |         |
| Ability 1|     |      |         |
| Ability 2|     |      |         |
| Performance|   |      |         |

**Comment by bachvudinh** [comment]
| Phase | Aspect | v0.2 | v0.3 |
|-------|--------|------|------|
| Phase 1 | Data Size |  |  |
|         | Data Source |  |  |
|         | Preprocessing |  |  |
|         | Model Architecture |  |  |
|         | Training Duration |  |  |
|         | Learning Rate |  |  |
|         | Batch Size |  |  |
|         | Data Task Type: |  |  |
|         | - Multiturn | | ✓ |
|         | - Speech | ✓ | ✓ |
|         | - Single-turn Text | ✓ | ✓ |
| Phase 2 | Data Size |  |  |
|         | Data Source |  |  |
|         | Preprocessing |  |  |
|         | Model Architecture |  |  |
|         | Training Duration |  |  |
|         | Learning Rate |  |  |
|         | Batch Size |  |  |
|         | Data Task Type: |  |  |
|         | - Multiturn | ✓ | ✓ |
|         | - Speech | ✓ | ✓ |
|         | - Single-turn Text | ✓ | ✓ |
| Phase 3 | Data Size |  |  |
|         | Data Source |  |  |
|         | Preprocessing |  |  |
|         | Model Architecture |  |  |
|         | Training Duration |  |  |
|         | Learning Rate |  |  |
|         | Batch Size |  |  |
|         | Data Task Type: |  |  |
|         | - Multiturn | ✓ | ✓ |
|         | - Speech | ✓ | ✓ |
|         | - Single-turn Text | ✓ | ✓ |
